src/infrastructure/validation/logger/ModuleID.java

```diff
@@ -1,16 +1,24 @@
-/**
- * 
- */
 package infrastructure.validation.logger;
 
 /**
+ * The ILogger interface depends on the ModuleID enum
+ * The ModuleID enum helps specify which module is calling the log
+ * Currently supported module names are
+ *   
```

Comment:
It is not intuitive to read "supported module name are" and then jump to enum definition while expecting it to be in the next line. It can help to specify that you have given the supported module names in the enum definition